### PR TITLE
Update dockerbuild.sh

### PR DESCRIPTION
**Pull request by tomchon**

### CODE REVIEW

**Review by stephenkgu — APPROVED** [review]

File: packaging/docker/dockerbuild.sh [hunk 1]
```diff
@@ -100,13 +110,65 @@ else
...
+  docker stop $(docker ps -aq)
+  docker rm $(docker ps -aq)
+fi
+
+if [  -n "$(pidof taosd)" ] ;then
+   echo "kill taosd "
+   kill -9 $(pidof taosd)
+fi
+
+if [ -n "$(pidof power)" ]  ;then
```
Comment (sangshuduo) [inline packaging/docker/dockerbuild.sh:128]: ?

**Review by sangshuduo — COMMENTED** [review]
a few comments

File: packaging/docker/dockerbuild.sh [hunk 2]
```diff
@@ -100,13 +110,65 @@ else
...
-docker build --rm -f "Dockerfile"  --network=host -t tdengine/tdengine-${dockername}:${version} "." --build-arg pkgFile=${pkgFile} --build-arg dirName=${dirName} --build-arg cpuType=${cpuTypeAlias}
+docker build --rm -f "${Dockerfile}"  --network=host -t tdengine/tdengine-${dockername}:${version} "." --build-arg pkgFile=${pkgFile} --build-arg dirName=${dirName} --build-arg cpuType=${cpuTypeAlias}
 docker login -u tdengine -p ${passWord}  #replace the docker registry username and password
 docker push tdengine/tdengine-${dockername}:${version}
 
-# set this version to latest version
-docker tag tdengine/tdengine-${dockername}:${version} tdengine/tdengine-${dockername}:latest
-docker push tdengine/tdengine-${dockername}:latest
+if [ -n "$(docker ps -aq)" ] ;then 
+  echo "delte docker process"
```
Comment (sangshuduo) [inline packaging/docker/dockerbuild.sh:118]: delete

File: packaging/docker/dockerbuild.sh (hunk 1 above)
Comment (tomchon) [reply packaging/docker/dockerbuild.sh:128]: because this server has some processes，like power 、taosd，which need to be killed

**Pull request merged**